docs/Spatial.md

```diff
@@ -1,6 +1,99 @@
+|[__phidget__.whenReady](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|function |This executes a function when the Phidgets RFID is ready to be used.|
+|[__phidget__.quit](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|N/A |This method requests a disconnect from the phidget board.  The disconnected event will be dispatched when the connection has been successfully disconnected. |
+|__phidget__.observeGyro|'gyro'|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) for the gyroscope on the __Spatial__ board. |
+|__phidget__.observeAcceleration|'accel'|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) for the acceleration sensors on the __Spatial__ board. |
+|__phidget__.observeMagneticField|'mag'|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) for the magnetic sensors, needed for the compass feature, on the __Spatial__ board. |
+
+##Data
+ 
+|Key|Data Type|Writable|Description|
+|---|---|---|---|
+|type|string|no|This is an identifier, Phidget GPS will always return 'PhidgetSpatial'|
+|accelerationMin|number|no|This returns the maximum negative acceleration that the desired axis will report.|
+|accelerationMax|number|no|This returns the maximum acceleration that the desired axis will report.|
+|angularRateMax|number|no|This returns the maximum angular momentum that the desired axis will report.|
+|angularRateMin|number|no|This returns the maximum angular momentum that the desired axis will report. |
+|dataRate|number|yes|This controls and reports current data rate at which data is being handled. In depth guide [here](http://www.phidgets.com/docs/1044_User_Guide) |
+|dataRateMax|number|no|This returns the maximum supported data rate.|
+|dataRateMin|number|no|is returns the minimum supported data rate.|
+|compassAxisCount|number|no|This returns the number of axes the PhidgetSpatial can measure magnetic field strength on. |
+|gyroAxisCount|number|no|This returns the number of axes the PhidgetSpatial can measure angular momentum on. |
+|accelerationAxisCount|number|no|This returns the number of axes the PhidgetSpatial can measure acceleration on. |
+|magneticFieldMax|number|no|This returns the maximum magnetic field that this axis will report.|
+|magneticFieldMin|number|no|This returns the maximum negative magnetic field that this axis will report.|
+|observeAcceleration|listener|no|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) the PhidgetSpacial for any acceleration changes and reports their activity.|
+|observeGyro|listener|no|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) the PhidgetSpacial for any changes in angular momentum and reports their activity.|
```

Comment:
This is not accurate, please look at how I described the observe function in GPS and RFID.

Also, this is a listener, it accepts a handler.